### ISSUE TITLE
Update paths

### ISSUE THREAD

**Issue by Matt-Butler** [cerner/terra-ui#75]
Update
```
'../../../packages/terra-site/src/PropsTable';
```
to
```
'terra-site/src/PropsTable';
```

This will help ensure our packages will work outside the monorepo.

Any relative paths can be replaced by adding the package as a devDependency and referencing it directly.

**Comment by bjankord** [comment]
Lets knock out #80 before we start this.

**Comment by bjankord** [comment]
Resolved by #100
